web/helpers.go

```diff
@@ -26,3 +29,23 @@ func publicError(c *gin.Context, statusCode int, err error) {
 		c.JSON(statusCode, models.NewJSONAPIErrorsWith(err.Error()))
 	}
 }
+
+func pagedResponse(
+	c *gin.Context,
+	name string,
+	size int,
+	page int,
+	resource interface{},
+	count int,
+	err error,
+) {
+	if err == orm.ErrorNotFound {
```

Comment:
Does 404 make sense for an index endpoint? I would expect an empty list.

Reply:
Doesn't make sense. Good catch.